library/ix-dev/enterprise/syncthing/questions.yaml

```diff
@@ -31,6 +31,74 @@ questions:
       $ref:
         - definitions/timezone
 
+  - variable: syncthingConfig
+    label: ""
+    group: Syncthing Configuration
+    schema:
+      type: dict
+      attrs:
+        # Boolean flags are currently
+        # not exposed to the user
```

Comment:
Flags added as **hidden** here because are already exist in the values.yaml to avoid issues with schema.